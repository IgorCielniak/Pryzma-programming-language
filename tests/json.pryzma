data = {"name": "John", "age": 30, "city": "New York"}
json_string = json_dump(data)
print json_string, "\n"
data2 = json_load(json_string)
print data2.name, "\n"
print data2.age, "\n"
print data2.city, "\n"

json_dump(data, "tests/test.json")
data3 = json_load("tests/test.json")
print data3.name, "\n"
print data3.age, "\n"
print data3.city, "\n"

remove_path("tests/test.json")
