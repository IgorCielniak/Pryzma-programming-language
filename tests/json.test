python Pryzma.py tests/json.pryzma
