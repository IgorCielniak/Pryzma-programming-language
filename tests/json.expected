{"name": "John", "age": 30, "city": "New York"}
John
30
New York
John
30
New York
